docs/tab-based-navigation.md

```diff
@@ -84,14 +87,61 @@ export default createBottomTabNavigator(
 );
 ```
 
-<a href="https://snack.expo.io/@react-navigation/tabs-with-icons-v3" target="blank" class="run-code-button">&rarr; Run this code</a>
+<a href="https://snack.expo.io/@sunnylqm/tabs-with-icons-v3" target="blank" class="run-code-button">&rarr; Run this code</a>
```

Comment:
I updated the code here to clean it up a little bit: https://snack.expo.io/@react-navigation/tabs-with-icons-v3

can you update the copy of it in these docs to match?